src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write code here
+const empList = document.querySelector('ul');
+const emp = [...document.querySelectorAll('li')];
+
+function sortList(list) {
+  list.sort((x, y) => {
+    const prev = +x.dataset.salary.replace(',', '').replace('$', '');
+    const next = +y.dataset.salary.replace(',', '').replace('$', '');
+
+    return next - prev;
+  });
+
+  return empList.append(...list);
+}
+
+function getEmployees(list) {
+  return [...list].map(employee => ({
```

Comment:
map method doesn't mutate the original list

```suggestion
  return list.map(employee => ({
```